src/dali_executor/CMakeLists.txt

```diff
@@ -28,9 +28,7 @@ set(
 
 include(${tritondalibackend_SOURCE_DIR}/cmake/dali.cmake)
 
-separate_arguments(EXTRA_DOWNLOAD_ARGS UNIX_COMMAND ${DALI_DOWNLOAD_EXTRA_OPTIONS})
```

Comment:
this was needed when the extra options were used directly in the COMMAND. It breaks now, because the options are pasted into the .in file.